.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+/Users/taylor.hottel/pivotassignments/cmd/marvelassignment/.env
```

Comment:
This only works on your machine. What's a better way to ignore `.env` for everybody?